src/coverlet.core/coverlet.core.csproj

```diff
@@ -7,15 +7,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="2.2.0" />
     <PackageReference Include="Mono.Cecil" Version="0.10.1" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.1" />
```

Comment:
I change this to 10 because Newtosolft has got a bug on 9.0.1 and cannot use TypeConverter in a correct way, this lead to exception during serialization of `CoveragePrepareResult` in testing.
MSbuild does not load this version but version 10.0.1 this is the reason why it works today(unexpected :smile:).